vowpalwabbit/metrics.cc

```diff
@@ -33,7 +33,7 @@ struct metrics_data
   size_t predicted_not_first = 0;
 };
 
-void list_to_json_file(std::string filename, std::vector<std::tuple<std::string, size_t>>& metrics)
+void list_to_json_file(vw& all, std::string filename, std::vector<std::tuple<std::string, size_t>>& metrics)
```

Comment:
is it worth passing a pointer to the `metrics` here instead of `all`?

Reply:
done